_layouts/rule-category.html

```diff
@@ -0,0 +1,8 @@
+---
+layout: single
+---
+{% assign filtered-rules = site.rules | where_exp: "item", "item.id contains page.category-id" %}
```

Comment:
- The categories are defined implicitly through rule identifiers. Another approach would be to explicitly define rule categories.
 - I could not find out, how to filter rules with a 'starts with' predicate but only with a 'contains' expression

Reply:
> * I could not find out, how to filter rules with a 'starts with' predicate but only with a 'contains' expression

I don't think this exists, but you can first _slice_ the first two characters and compare them directly. See https://shopify.github.io/liquid/filters/slice/

> * The categories are defined implicitly through rule identifiers. Another approach would be to explicitly define rule categories.

I think that would be the most clean approach.